src/message.rs

```diff
@@ -0,0 +1,12 @@
+use super::schema::message;
+
+#[derive(Queryable)]
+#[derive(Insertable)]
+#[table_name="message"]
+pub struct Message {
+    pub id: i64,
+    pub content: String,
+    pub guild_id: i64,
+    pub author_id: i64,
+    pub channel_id: i64,
+}
```

Comment:
These types are wrong. Discord snowflakes are always `u64`, not signed.
All of these should be changed.